app/views/users/_user_info.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="user-info-container">
+    <div class="bio">
+        <h2>Bio</h2>
+        <p><%= user.bio %></p>
+    </div>
+    <% counter = 0 %>
+    <% user.posts.each_with_index do |post, idx| %>
+        <% if counter < 2 %>
+            <div class="post">
+            <h2>Post: <%= post.title %></h2>
+            <p><%= post.text %></p>
+            </div>
+            <% counter += 1 %>
+        <% else %>
+            <% break %>
+        <% end %>
```

Comment:
- The user#show page should show the last three posts, it is showing just two. Please 🙏🏽 use the method `recent_three_posts` from your model to be able to see the last three posts.

![image](https://github.com/wacoo/blog-front-ruby-rails/assets/58642949/3aaa3bab-c743-411e-a0f5-aa1ac95f2f84)